doc/dev/release.md

```diff
@@ -307,11 +308,28 @@ OPERATORSUM="$(sha256sum operator-sdk | cut -d ' ' -f 1)"
 brew bump-formula-pr --strict --url=$OPERATORSDKURL --sha256=$OPERATORSUM operator-sdk
 ```
 
-Note: If there were any changes made to the CLI commands, make sure to look at the existing tests, in case they need updating.
+**NOTE**  If there were any changes made to the CLI commands, make sure to look at the existing tests, in case they need updating.
 
 You've now fully released a new version of the Operator SDK. Good work! Make sure to follow the post-release steps below.
 
-### (Post-release) Updating the operator-sdk-samples repo
+### 6. Create JIRA tasks for the post-release actions
+
+After we do the release the following post-release actions are required to attend the users needs. So, create a JIRA for the post release actions  and ping them in the channel in order to allow other sign themselves to help with.
+
+**NOTE** You are free to working on in the post-release tasks if you wish as well. 
+
+## Post-release steps
+
+### 1. Update the Upgrade version guide document and Go Memcached Sample
+
+1. The [Upgrade version guide][version-upgrade-guide] should be updated with the steps required for the users are able to migrate their project. Then, in order to know the steps and ensure that they will work upgrade the [Go Memcached Sample][go-memcached-sample].
+1. Then, update the GitHub Release Notes as:
+
+```
+**NOTE** See how to upgrade your project to the version v1.3.0 by checking the [Version Upgrade Guide](../migration/version-upgrade-guide.md#v1.3.0)
+```
```

Comment:
@joelanford 
I will wait for we merge this one, then I will do these updates for all versions.